recipes/swashes/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "SWASHES" %}
+{% set version = "1.03.00" %}
+{% set renater_num = "4900" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourcesup.renater.fr/frs/download.php/file/{{ renater_num }}/{{ name }}-{{ version }}.zip
+  sha256: 05adfe317369e03821aaf424b25688fe416b055b88ddee78702769760689ca0d
+  patches:
+    - swashes.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
this probably implicitly uses make on unix

Reply:
Done in 8b6fd97